definitions/torrentz2.yml

```diff
@@ -0,0 +1,53 @@
+---
+  site: torrentz2
+  name: Torrentz2
+  language: en-us
+  links:
+    - https://torrentz2.eu/
+
+  caps:
+    categories:
+      "Video Tv": TV
+      "Video": Movies
+      "Video Movie HD": Movies/HD
+
+    modes:
+      search: [f]
+      tv-search: [f, season, ep]
+
+  search:
+    path: feed 
+    inputs:
+       f: "{{ .Query.Keywords }}"
+    rows:
+      selector: "rss channel item"
+    fields:
+      title:
+        selector: title
+      download:
+        selector: description
+        filters:
+          - name: regexp
+            args: "Hash: (\\w+)"
+      category:
+        selector: category
+      date:
+        selector: pubDate
+        filters:
+          - name: dateparse
+            args: "Mon, 02 Jan 2006 15:04:05 -0700"
```

Comment:
this doesn't cover all cases.
day and hours seem to be used without leading zeros.
The problem is that go doesn't support the 24 hour format without leading space.
A workaround would be to use "Mon, 2 Jan 2006 15:04:05 -0700" but use 9 replace filters like [" 1:", " 01:"] before that to convert the 12H format to 24H.